documentation/general/dash-sonic-hld.md

```diff
@@ -265,6 +266,73 @@ It is possible that a given packet can get a hit in route table and/or mapping t
 
   ![dash-outbound-meter](./images/dash-hld-outbound-meter-pipeline.png)
 
+## 2.5 FastPath
+
+This section captures the Sonic-Dash specifics of FastPath use-case. Detailed document on FastPath is captured here (![FastPath](https://github.com/sonic-net/DASH/blob/main/documentation/load-bal-service/load-balancer-v3.md))
```

Comment:
This link is not a image, so we will need to remove the "!" from "![FastPath]".

Reply:
addressed